db/system_keyspace.cc

```diff
@@ -2767,7 +2767,7 @@ future<service::topology> system_keyspace::load_topology_state() {
                     "load_topology_state: node {} in normal state but missing ring slice", host_id));
             }
         } else if (nstate == service::node_state::left) {
-            ret.left_nodes.emplace(host_id);
+            map = &ret.left_nodes;
```

Comment:
Alternatively, we can replace all mentions of the node by a null uuid, and teach everyone to ignore it.

Reply:
No, you explain why that doesn't work.